http-client/pom.xml

```diff
@@ -158,6 +153,12 @@
             <scope>runtime</scope>
         </dependency>
 
+        <dependency>
```

Comment:
Merge `Tests for streaming execution` into commit which adds support for streaming responses.